sormas-e2e-tests/src/main/java/org/sormas/e2etests/common/StringUtils.java

```
@@ -0,0 +1,18 @@
+package org.sormas.e2etests.common;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringUtils {
+
+  public static String getMatchedGroupByIndexFromAString(
+      String string, String regex, int groupIndex) {
+    Pattern pattern = Pattern.compile(regex);
+    Matcher matcher = pattern.matcher(string);
+    if (matcher.find()) {
+      return matcher.group(groupIndex);
+    } else {
+      return null;
```

Please throw an exception instead of returning null.